apidoc/Titanium/UI/Window.yml

```diff
@@ -1147,7 +1149,7 @@ properties:
     description: |
         This was separated from the <Titanium.UI.Window.androidback> event. You need to define this
         callback if you explicitly want to override the back button behavior.
-    type: Callback<Object>
+    type: Callback<void>
```

Comment:
yeah I believe it is in our doc format. `Callback` is synonymous with `Function`. As to the ` void` arg, well that is seen as ok in the validation. Otherwise, I'm not sure how to define that the function takes no arguments. I think the generators are ok with with `void` arg